File: Python/1117 - Score Validation.py
notas_validas=0
total_notas=0
while notas_validas<2:
    n=float(input())
    if n<0 or n>10:
        print("nota invalida")
    else:
        total_notas+=n
        notas_validas+=1
media=total_notas/2
print(f"media = {media:.2f}")

File: Java/1117 - Score Validation.java
import java.util.Scanner;
public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int notasValidas = 0;
		double media = 0,soma = 0;
		while (notasValidas < 2) {
		    double x = sc.nextDouble();
		    if (x>10 || x<0) 
		        System.out.println("nota invalida");
            else {
            soma += x;
            notasValidas++;
            }
		}
		media = soma / 2.0;
		System.out.printf("media = %.2f\n",media);
		sc.close();
	}
}
